src/braket/experimental/autoqasm/instructions/gates.py

```diff
@@ -0,0 +1,456 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+
+"""Quantum gates, unitary instructions, that apply to qubits.
+"""
+from braket.experimental.autoqasm import program
+
+from .qubits import _qubit
+
+
+def ccnot(control_0: int, control_1: int, target: int) -> None:
```

Comment:
I believe all of the qubit args should have type hint `QubitIdentifierType` rather than `int`.